src/util/interfaces.ts

```diff
@@ -0,0 +1,38 @@
+export interface AttachmentProps {
+  id: number;
+  file: string;
+  created: string;
+}
+export interface MessageProps {
+  id?: number;
```

Comment:
id is optional?

Reply:
Good point. I got some time after work to do this 👍

Reply:
Yeah so when the `last_message` attribute in a `chat` objects may not have an ID if the message is blank.

This is something we can patch with the API later but now it's optional